proposals/0003.md

```diff
@@ -0,0 +1,83 @@
+* Title: *Licensing and DCO*
+* Champion: *@FintanH/fintan.halpenny@gmail.com@*
+* Conversation:
+    * https://radicle.community/t/radicle-licensing-model/282
+    * https://radicle.community/t/were-implementing-a-developer-certificate-of-origin-dco/303
+* References:
+    * [Linux Kernel DCO][dco]
+    * [GPLv3 License][gplv3]
+
+# Summary
+
+The decision here is to implement our software under the license of
+GPLv3+linking exception. This also means that outside contributors must sign off
+on their commits to our software, as described by our Developer Certificate of Origin (DCO).
+
+# Explanation
+
+Most of the explanation of why we are doing this is recorded in the
+`radicle.community` links above. What this means is that all repositories under
+`radicle-dev` must comply with the following sections.
+
+# Recommendation
+
+In this section we include the templates that need to be included and also other
+recommendations to make the licensing experience as smooth as possible for both
+contributors and core developers.
+
+## License File
+
+Every repository MUST include a `LICENSE` file which corresponds to this
+[file][license-file].
```

Comment:
The license file actually needs to include the linking exception text (at the top?).

Reply:
There are also SPDX headers, which indicate the license in a single line. Not sure about the legal might of these though